promotion/replication/replication.ipynb

```diff
@@ -6,110 +6,355 @@
    "metadata": {},
```

Comment:
<p>I know, this was already there by me. But could you write this in markup and use a hyperlink for the websites?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/ruspy/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>